### DISCUSSION TITLE
secret scanning and email alerts

### DISCUSSION THREAD

**Discussion by Timmmy-nlb** [community/community#34673] [Code Security]
In the documentation it is described that Repo-Admin and Org-Owner should get a mail if there is an alarm.

see:
[https://docs.github.com/en/enterprise-cloud@latest/code-security/secret-scanning/about-secret-scanning#about-secret-scanning-alerts](https://docs.github.com/en/enterprise-cloud@latest/code-security/secret-scanning/about-secret-scanning#about-secret-scanning-alerts)

But I never got a mail for this as an Org-Owner.
Is there still a requirement missing or is it just broken?

**Comment by 15MariamS** [comment]
Hi @Timmmy-nlb, 

You should receive an email notification if you are _watching_ the repository and have [enabled notifications](https://docs.github.com/en/enterprise-cloud@latest/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#configuring-your-watch-settings-for-an-individual-repository) for security alerts or for all the activity on the repository.

Can you confirm if you are watching the repos in your org that you expect to receive the notifications on?

**Reply by Timmmy-nlb** [reply]
Hello @15MariamS ,
thank you for the reply.

No, I had left everything in the default ("Participating and @mentions").
However, I find this default setting rather bad for enterprise owners (with many Orgs and Repos).

I would wish that there could be an option (policy), which always sends the security alerts in the default owner of an organization - but this can be deactivated manually.

So the owner (which have with us a professional supervision) must always manually (even new) Repos the watch (and I must not forget to do so).
